deployment/scripts/chainspec/dev/state_config.json

```diff
@@ -124,10 +124,10 @@
       "contract_id": "7777777777777777777777777777777777777777777777777777777777777777",
       "code": "9000000994318e6e453f30e85bf6088f7161d44e57b86a6af0c955d22b353f91b2465f5e6140000a504d00205d4d30001a4860004945048076440001240400005050c0043d51345024040000",
       "salt": "1bfd51cb31b8d0bc7d93d38f97ab771267d8786ab87073e0c2b8f9ddc44b274e",
-      "tx_id": null,
-      "output_index": null,
-      "tx_pointer_block_height": null,
-      "tx_pointer_tx_idx": null
+      "tx_id": "0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
I'm curios why zero tx id is at the end=D

Reply:
I felt the contract was special 😅